readme.md

```diff
@@ -8,65 +8,399 @@
 [![Backers][backers-badge]][collective]
 [![Chat][chat-badge]][chat]
 
-**remark** is a markdown processor built on [micromark][] powered by
-[plugins][] part of the [unified][] collective.
+**remark** is a tool that transforms markdown with plugins.
+These plugins can inspect and change your markup.
+You can use remark on the server, the client, CLIs, deno, etc.
 
-## Intro
-
-**remark** is [the world’s most popular markdown parser][popular]!
-And it does so much more than parse: it inspects (check, lint) and transforms
-(generate, compile) markdown too!
-
-Everything is possible with plugins, such as [checking markdown code
-style (`remark-lint`)][remark-lint], [transforming safely to React
-(`remark-react`)][remark-react], [adding a table of
-contents (`remark-toc`)][remark-toc], or [compiling to man pages
-(`remark-man`)][remark-man].
-
-Internally, remark now uses [micromark][], a new, fast, and tiny CommonMark
-compliant markdown tokenizer.
-It can be GFM compliant with [`remark-gfm`][remark-gfm].
+## Feature highlights
 
-Finally, remark is part of the [unified][website] [collective][governance].
-Learn more about us:
+*   [x] **[popular][]** (world’s most popular markdown parser)
+*   [x] **[compliant][syntax]** (100% to CommonMark, 100% to GFM with a plugin)
+*   [x] **[plugins][]** (150+ plugins you can pick and choose from)
+*   [x] **[ASTs][syntax-tree]** (inspecting and changing content made easy)
 
-*   Visit [`unifiedjs.com`][website] and peruse its [Learn][] section for an
-    overview
-*   Read [unified][]’s readme for a technical intro
-*   Browse [awesome remark][awesome] to find out more about the ecosystem
-*   Follow us on [Twitter][] to see what we’re up to
-*   Check out [Contribute][] below to find out how to help out
-
-## Packages
-
-This repository contains the following packages:
+## Intro
 
-*   [`remark-parse`][parse] — Parse markdown to syntax trees
-*   [`remark-stringify`][stringify] — Serialize syntax trees to markdown
-*   [`remark`][api] — Programmatic interface with both `remark-parse` and `remark-stringify`
-*   [`remark-cli`][cli] — Command line interface wrapping `remark`
+remark is a very popular ecosystem of plugins that work with markdown as
+structured data, specifically ASTs (abstract syntax trees).
+ASTs make it easy for programs to deal with markdown.
+We call those programs plugins.
+Plugins inspect and change trees.
+You can use the many existing plugins or you can make your own.
```

Comment:
Repeating back the keypoints in the order they appear:
1. remark is popular
2. remark is for markdown
3. remark uses ASTs
4. ASTs make it easy to edit markdown
5. Plugins can also edit things
6. There are a lot of plugins

Is that the order we want to present things?
Maybe:

1. remark is for markdown
2. remark uses ASTs and plugins
3. ASTs and plugins make editing easy
4. remark has a large community
5. community includes many remade plugins

?

Reply:
I’m not sure what this would mean practically?
I don’t think remark really has a large community. I use popular here to mention that it is used a lot.

Reply:
> I don’t think remark really has a large community. I use popular here to mention that it is used a lot.

Adopters aren't part of the community?

Reply:
Who do you mean by adopters?

*Community* to me sounds like folks actively interacting with repos, discussions, etc, and I don’t think we have much of that.

Reply:
Adopters as in folks downloading and using remark, maybe opening an issue or a discussion, but not yet a maintainer.

Reply:
So “user” in [this chart](https://github.com/unifiedjs/collective/blob/main/members.md), right?
There’s a divide there: folks that choose to put a markdown package in their `package.json`, and folks that use packages that somewhere include remark.
remark, compared to others, has few of those adopters/explicit users: https://github.com/remarkjs/remark/network/dependents?dependent_type=PACKAGE&package_id=UGFja2FnZS0xNDIzNzIzMw%3D%3D, https://github.com/markedjs/marked/network/dependents?dependent_type=PACKAGE&package_id=UGFja2FnZS0xMzQzNjIxNw%3D%3D, https://github.com/markdown-it/markdown-it/network/dependents?dependent_type=PACKAGE&package_id=UGFja2FnZS0xODE4NTUwMQ%3D%3D
But the most of implicit users, by download counts.
I don’t think it’s fair to say we have the most vibrant community here, but it is fair to say we’re the most popular.

Reply:
but this is sort of besides the point of your earlier layout of a rewrite?